retroshare-gui/src/gui/settings/rsharesettings.cpp

```diff
@@ -1150,9 +1153,23 @@ void RshareSettings::setWebinterfaceEnabled(bool enabled)
     setValueToGroup("Webinterface", "enabled", enabled);
 }
 
+QString RshareSettings::getWebinterfaceFilesDirectory()
+{
+#ifdef WINDOWS_SYS
+    return valueFromGroup("Webinterface","directory","data/webui/").toString().toStdString();
+#endif
+    return valueFromGroup("Webinterface","directory","/usr/share/retroshare/webui/").toString();
```

Comment:
again those hardcoded values won't work on all platforms/distro, use the method we have somewhere to get data directory